action.yml

```diff
@@ -50,7 +50,7 @@ runs:
   steps:
     - id: helm-install
       name: Install a Helm chart
-      uses: docker://public.ecr.aws/l7o7z1g8/actions/helm-install-action:1.1.4
+      uses: docker://public.ecr.aws/l7o7z1g8/actions/helm-install-action:${{ action.scm.sha }}
       # TODO: use interpolated tag once we aligned on the release process
```

Comment:
I would remove these comments as they shouldn't be relevant with the sha based release flow